src/lib/stellar-guard.ts

```diff
@@ -0,0 +1,49 @@
+import { Server, Transaction } from "stellar-sdk"
+
+const STELLARGUARD_TRANSACTION_ENDPOINT_MAINNET = "https://stellarguard.me/api/transactions"
+const STELLARGUARD_TRANSACTION_ENDPOINT_TESTNET = "https://test.stellarguard.me/api/transactions"
+const STELLARGUARD_PUBLIC_KEY = "GCVHEKSRASJBD6O2Z532LWH4N2ZLCBVDLLTLKSYCSMBLOYTNMEEGUARD"
+
+interface AccountSignerWithKey {
+  public_key: string
+  key: string
+  weight: number
+}
+
+export async function isStellarGuardProtected(horizon: Server, accountPubKey: string) {
+  const account = await horizon.loadAccount(accountPubKey)
+
+  const signers = account.signers as AccountSignerWithKey[] // the name of the property is key and not public_key
```

Comment:
Good catch! 👍

Has already been fixed in the latest Stellar SDK. About time to tackle #460, I suppose... ;)